README.md

```diff
@@ -67,13 +92,15 @@ If you are on certain distribution of Linux, you will need to alias `fdfind` to
 
 ### Using custom key bindings
 
-Each function is assigned mnemonic key bindings by default (see screenshots above) in [conf.d/fzf.fish][]. However, if you would like to customize them, you can prevent the default key bindings from executing by setting `fzf_fish_custom_keybindings` as a [universal variable][]. You can do this by running
+If you would like to customize the key bindings, first, prevent the default key bindings from executing by setting `fzf_fish_custom_keybindings` as an [universal variable][]. You can do this with
```

Comment:
Now that the key bindings and their mnemonics are more prominent, don't need to reference them; assume the reader is already aware of them.